variables.tf

```diff
@@ -0,0 +1,119 @@
+variable "region" {
+  description = "The region used to launch this module resources."
+  type        = string
+  default     = ""
+}
+
+variable "profile" {
+  description = "The profile name as set in the shared credentials file. If not set, it will be sourced from the ALICLOUD_PROFILE environment variable."
+  type        = string
+  default     = ""
+}
+variable "shared_credentials_file" {
+  description = "This is the path to the shared credentials file. If this is not set and a profile is specified, $HOME/.aliyun/config.json will be used."
+  type        = string
+  default     = ""
+}
+
+variable "skip_region_validation" {
+  description = "Skip static validation of region ID. Used by users of alternative AlibabaCloud-like APIs or users w/ access to regions that are not public (yet)."
+  type        = bool
+  default     = false
+}
+
+// Market Product
+variable "product_name_prefix" {
+  description = "The name prefix of Market Product"
+  type        = string
+}
+
+// ECS instance
+variable "create_instance" {
+  description = "Whether to create ecs instance."
+  type        = bool
+  default     = true
+}
+
+variable "ecs_instance_name" {
+  description = "The name of ECS Instance."
+  type        = string
+  default     = "TF-Wordpress"
+}
+variable "ecs_instance_password" {
+  description = "The password of ECS instance."
+  type        = string
+  default     = ""
+}
+variable "image_id" {
+  description = "The image id used to launch one ecs instance. If it is '', Alibaba Cloud market place image will be used"
```

Comment:
The image id used to launch one ecs instance. If not set, a fetched market place image by product_keyword will be used.